### DISCUSSION TITLE
Notification  if miner disconect.

### DISCUSSION THREAD

**Discussion by totom9000** [coinfoundry/miningcore#1346] [General]
Hey  have some miners asking if we culd add notification  that send them email if miner that mines to their acc disconect.

And was thinking  this isnt stupid many pools have this and there is a good develuped  admin notification  here already  so belive this maybe isnt so hard to add.

Adding a picture howe other pools do this. 
![Screenshot_20220808-175559_Chrome](https://user-images.githubusercontent.com/44748488/183461753-619bc9f3-49da-4de8-a2fb-b49f09d65e5c.jpg)

**Comment by seperatis** [comment]
Everyone, please STOP 🛑 opening up issues for feature requests and General Questions. 

✅ That's why we have a Discussion tab!

Please, ONLY open a new Issue if it's a BUG or issue related directly to MC software itself.
